docs/index.md

```diff
@@ -1,12 +1,12 @@
 ---
 # generated by https://github.com/hashicorp/terraform-plugin-docs
-page_title: "dbt-cloud Provider"
+page_title: "dbt Provider"
 subcategory: ""
 description: |-
   
 ---
 
-# dbt-cloud Provider
+# dbt Provider
```

Comment:
I guess this is the downside, if it's now dbt provider, but not the end of the world

Reply:
Yes, I think that getting it to be dbt-cloud but still work with templating might require renaming a few things.

Reply:
Yeah, I think that's something I was partly trying to solve by moving the the new framework where the name is in a metadata object